src/main/java/com/github/tomakehurst/wiremock/client/ResponseDefinitionBuilder.java

```diff
@@ -90,6 +90,19 @@ public static ResponseDefinitionBuilder like(ResponseDefinition responseDefiniti
     return builder;
   }
 
+  private static ResponseDefinitionBuilder createProxyResponseDefinitionBuilder(
```

Comment:
I am not sure this new method is used now you have merged in master.  Can you check and re-apply your previous change if you still want it.  If not then we should remove this method

Reply:
This method is used in public static ResponseDefinitionBuilder like(ResponseDefinition responseDefinition) as I extracted method from thaat method

Reply:
Hi @Shivang-X Are you sure this is the case ?  When you originally put the PR together this method was used and the code extracted from the `like` method.  You then merged in the master branch and this put the extracted code back to what it was before but still with the extracted method.  This means the method is currently unused.  If you run the tests you will see they are failing because this method is unused.